templates/vue/src/components/lightbox/WpPostMedia.vue

```diff
@@ -39,6 +44,13 @@ export default {
     this.title = post.title
     this.content = post.content
     this.$emit("complete")
+    this.$emit("load")
+
+    this.$nextTick(() => {
+      const post = this.$refs.post
+      this.scrollHeight = post.scrollHeight
+      this.clientHeight = post.clientHeight
```

Comment:
This has to be done anytime the lightbox's height changes. In places like the accordion that can change often, this gets slightly complex.

Reply:
Ok, but this is in `wpPostMedia` - should we add this to Accordion Media as well? And fire it off everytime the container is resized? I know in jQuery you have a `resize` event on elements - not sure what's the Vue equivalent - or we can just use vanilla JS

Reply:
Sorry if I wasn't clear. I meant that this same code is copied and reused in all media types that require a scrollbar. I highlighted this to show the complexity that implementing this custom scrollbar introduces - if (in your opinion) this added code isn't a problem, then I wouldn't worry about it.